whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -2773,8 +2773,7 @@ class PostgreSQLComponent {
                 def allow = JsonLd.ALLOW_LINK_TO_DELETED + (jsonld?.cascadingDeleteRelations() ?: Collections.EMPTY_SET)
                 def referencedBy = followDependers(identifier, allow)
```

Comment:
Note to selves: the actual problem is that we are using followDependers though we only care about docs linking directly to the doc being deleted? But there are only links when we block the delete so probably fine.

Changed in https://github.com/libris/librisxl/commit/230fe6f4f515cb11b1b2619977eeee5a84329b97